### PR TITLE
[ trivial ] Fix typo & misspelled function param

### PR DESCRIPTION
**Pull request by skykongkong8**
- intY -> incY (incremental index of Y)
- __scopy_kernel() only works for interested SIMD register width

**Self evaluation:**
1. Build test:     [X]Passed [ ]Failed [ ]Skipped
2. Run test:     [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by EunjuYang — APPROVED** [review]
LGTM

File: nntrainer/tensor/blas_interface.h [hunk 1]
```diff
@@ -265,23 +265,23 @@ void scopy(const unsigned int N, const void *X, const int incX, void *Y,
...
  */
 void scopy(const unsigned int N, const float *X, const int incX, float *Y,
-           const int intY);
+           const int incY);
 /**
  * @brief     copy function : Y = X
  * @param[in] N number of elements in X
  * @param[in] X uint8_t * for Vector X
  * @param[in] Y uint8_t * for Vector Y
  */
```
Comment (EunjuYang) [inline nntrainer/tensor/blas_interface.h:274]: What about adding explanation on `incX` and `incY`?

**Review by myungjoo — APPROVED** [review]

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Comment by SeoHyungjun** [comment]
LGTM

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by djeong20 — APPROVED** [review]

**Pull request merged**